src/js/tabs/pid_tuning.js

```diff
@@ -106,7 +106,8 @@ TABS.pid_tuning.initialize = function (callback) {
             $('.pid_tuning input[name="rc_expo"]').attr("rowspan", "3");
         }
 
-        if (semver.gte(CONFIG.apiVersion, "1.16.0")) {
+        $('.vbatpidcompensation').toggle(semver.lt(CONFIG.apiVersion, API_VERSION_1_44));
+        if ((semver.gte(CONFIG.apiVersion, "1.16.0")) && (semver.lt(CONFIG.apiVersion, API_VERSION_1_44))) {
             $('input[id="vbatpidcompensation"]').prop('checked', ADVANCED_TUNING.vbatPidCompensation !== 0);
```

Comment:
I don't understand exactly why this code does:
- Before it verifies if bigger than 1.16 (I suppose is the first version that appeared the feature) for the property. But it does not modify the visibility.
- Now it shows if lower than 1.44 but does not verify if bigger than 1.16 for the visibility. But it does for the property.

I suppose it must verify if bigger than 1.16 for the visibility. Is that true? 

About the property, we can verify it or not. Lately I prefer to verify only the visibility for simplicity in the code. The property will take the default value defined in MSP, but it will be not visible so no problem with it. But this file verifies it for all of them, so maybe better to maintain.

Reply:
@McGiverGim , it seems that this feture only supports a pre 3.0 version, the msp code is not very clear and defines it for versions smaller than 1.20 and greater than 1.20, I don't know if there was a need to check the visibility, do you think we should hide it to versions before 1.16?
```
            // only supports 1 version pre bf 3.0
            else {
                buffer.push16(ADVANCED_TUNING.rollPitchItermIgnoreRate)
                   .push16(ADVANCED_TUNING.yawItermIgnoreRate)
                   .push16(ADVANCED_TUNING.yaw_p_limit)
                   .push8(ADVANCED_TUNING.deltaMethod)
                   .push8(ADVANCED_TUNING.vbatPidCompensation);
            }
```

Reply:
I think it must be coherent between the hide and the property. It has no sense to show it but don't update the property from the variables. I think we can simply remove the condition for the property, as I say makes a cleaner code and it does not hurt because it will use the default property value. Something like:
```
const vbatpidcompenastionenabled = semver(what conditions we decide);
$('.vbatpidcompensation').toggle(vbatpidcompenastionenabled);
$('input[id="vbatpidcompensation"]').prop('checked', ADVANCED_TUNING.vbatPidCompensation !== 0);
```
Of at least:
```
if (semver("what conditions we decide)") {
    $('.vbatpidcompensation').show();
    $('input[id="vbatpidcompensation"]').prop('checked', ADVANCED_TUNING.vbatPidCompensation !== 0);
} else {
    $('.vbatpidcompensation').hide();
}
```